services/game-session-service/src/main/resources/redis/restore_pending_projection.lua

```
@@ -0,0 +1,199 @@
+    end
+  end
+end
+
+local pendingPayloadIndex = pendingPayloadStartIndex
+for index = 1, pendingCount do
+  redis.call('LREM', pending, 0, ARGV[pendingPayloadIndex])
+  pendingPayloadIndex = pendingPayloadIndex + 1
+end
+
+local sealedPayloadIndex = sealedPayloadStartIndex
+for index = 1, sealedCount do
```

Luacheck (1.2.0)

[warning] unused loop variable 'index'

(W213)